gslib/commands/acl.py

```diff
@@ -340,6 +346,50 @@ class AclCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    sub_command = self.args.pop(0)
+    if sub_command == 'get':
+      if StorageUrlFromString(self.args[0]).IsObject():
+        command_group = 'objects'
+      else:
+        command_group = 'buckets'
+      gcloud_storage_map = GcloudStorageMap(gcloud_command=[
+          'alpha', 'storage', command_group, 'describe', self.args[0], '--format=json[acl]'
+      ],
+                                            flag_map={})
+      self.args = self.args[:-1]
```

Comment:
I think we can keep the last argument, which is `url` according to http://cloud/storage/docs/gsutil/commands/acl

```
 gcloud_storage_map = GcloudStorageMap(gcloud_command=[
          'alpha', 'storage', command_group, 'describe', '--format=json[acl]'
      ],
                                            flag_map={})
```

Removing `self.args[0]` from the current translation should cause the `url` to get translated and added to the end of the command, resulting in `... describe --format=json[acl] url`